### ISSUE TITLE
Replace Heapster with the metrics-apiserver

### ISSUE THREAD

**Issue by mrIncompetent** [kubermatic/kubermatic#852]
We deploy heapster currently with the addon manager.
The metrics server should be deployed the same way.

We need to investigate if apiserver aggregation even works in our architecture.

**Comment by metalmatze** [comment]
Duplicate of #708

**Comment by metalmatze** [comment]
Blocked by #871

**Comment by scheeles** [comment]
Isn't `hpa` not depending on heapster anymore?

**Comment by mrIncompetent** [comment]
Afik no, the whole metrics topic got abstracted away.
See https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/

**Comment by mrIncompetent** [comment]
Closing in favor of #871

**Comment by metalmatze** [comment]
We still need to replace heapster with metrics-server? Why is this closed? :eyes: :thinking: 

Here's the official deprecation issue:
https://github.com/kubernetes/heapster/pull/2022

**Comment by mrIncompetent** [comment]
Referenced the wrong issue.
Closed in favor of the original replace issue: https://github.com/kubermatic/kubermatic/issues/708
